doc/user/Frontends.md

```diff
@@ -10,34 +10,34 @@ For the purposes of this document and in many OpenTripPlanner development discus
 
 Broadly speaking there are two kinds of frontends for OTP: debug frontends and production frontends. 
 
-**Debug frontends** are included in the main OpenTripPlanner repository, and are intended to work "out of the box" with little to no configuration. They are served by OTP itself or a simple local web server, or set up such that a single deployment is usable by anyone via a content delivery network (CDN). The primary purpose of debug frontends is to allow OTP developers (as well as people evaluating OpenTripPlanner for their organization) to observe, test, and debug an OTP instance they're working on. Debug frontends will expose some internal details of OTP operation such as graph nodes and edges, traversal permissions, or transit data entity identifiers. Their primary role is as a development and deployment tool.
+A **Debug frontend** is included in the main OpenTripPlanner repository, and it is intended to work "out of the box" with little to no configuration. The old debug frontend of OTP has been permanently removed and replaced by the new debug UI. It is served by OTP itself or a simple local web server, or set up such that a single deployment is usable by anyone via a content delivery network (CDN). The primary purpose of debug frontends is to allow OTP developers (as well as people evaluating OpenTripPlanner for their organization) to observe, test, and debug an OTP instance they're working on. Debug frontends will expose some internal details of OTP operation such as graph nodes and edges, traversal permissions, or transit data entity identifiers. Their primary role is as a development and deployment tool.
 
 On the other hand, **production frontends** are intended to be a component of larger public-facing deployments of OTP, with a more polished appearance and user experience. They generally do not work "out of the box", and require a significant amount of configuration and coordination with external components such as map tile servers and geocoders. They are designed to be components of a large professionally maintained and managed OTP deployment, and present a simpler view of OpenTripPlanner options and results to a non-technical audience of end users. 
 
 ## Debug Frontends
 
-The main OpenTripPlanner repository currently contains two debug web frontends: 
+The main OpenTripPlanner repository currently contains one debug web frontend: 
```

Comment:
```suggestion
```

Reply:
I merged this into the next suggestion, see below

Reply:
I removed this line